source/fuzz/fuzzer_pass_add_dead_continues.cpp

```diff
@@ -32,15 +32,41 @@ void FuzzerPassAddDeadContinues::Apply() {
   // Consider every block in every function.
   for (auto& function : *GetIRContext()->module()) {
     for (auto& block : function) {
+      // Get the label id of the continue target
+      // of the innermost loop.
+      auto continue_block_id =
+          block.IsLoopHeader()
+              ? block.ContinueBlockId()
+              : GetIRContext()->GetStructuredCFGAnalysis()->LoopContinueBlock(
+                    block.id());
+
+      // This transformation is not applicable
+      // if current block is not inside a loop.
```

Comment:
Nit: I like to use up as much of the 80-char whitespace as possible with comments (unless there's a reason for a line break), so I would change this to:

      // This transformation is not applicable if current block is not inside a
      // loop.

Reply:
Fixed in a04b185.